diff --git a/node_modules/ion7-calendar/dist/index.d.ts b/node_modules/ion7-calendar/dist/index.d.ts
index 788ae87..37dd917 100644
--- a/node_modules/ion7-calendar/dist/index.d.ts
+++ b/node_modules/ion7-calendar/dist/index.d.ts
@@ -2,6 +2,7 @@ export * from './calendar.model';
 export { CalendarModal } from './components/calendar.modal';
 export { CalendarWeekComponent } from './components/calendar-week.component';
 export { MonthComponent } from './components/month.component';
+export { MonthPickerComponent } from './components/month-picker.component';
 export { DEFAULT_CALENDAR_OPTIONS } from './services/calendar-options.provider';
 export { CalendarComponent } from './components/calendar.component';
 export { CalendarModule } from './calendar.module';
diff --git a/node_modules/ion7-calendar/dist/index.js b/node_modules/ion7-calendar/dist/index.js
index d3490e8..853ba0a 100644
--- a/node_modules/ion7-calendar/dist/index.js
+++ b/node_modules/ion7-calendar/dist/index.js
@@ -22,6 +22,8 @@ var calendar_week_component_1 = require("./components/calendar-week.component");
 Object.defineProperty(exports, "CalendarWeekComponent", { enumerable: true, get: function () { return calendar_week_component_1.CalendarWeekComponent; } });
 var month_component_1 = require("./components/month.component");
 Object.defineProperty(exports, "MonthComponent", { enumerable: true, get: function () { return month_component_1.MonthComponent; } });
+var month_picker_component_1 = require("./components/month-picker.component");
+Object.defineProperty(exports, "MonthPickerComponent", { enumerable: true, get: function () { return month_picker_component_1.MonthPickerComponent; } });
 var calendar_options_provider_1 = require("./services/calendar-options.provider");
 Object.defineProperty(exports, "DEFAULT_CALENDAR_OPTIONS", { enumerable: true, get: function () { return calendar_options_provider_1.DEFAULT_CALENDAR_OPTIONS; } });
 var calendar_component_1 = require("./components/calendar.component");
